src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -20,33 +19,31 @@ const resolveToggleLink = (link, featureToggles) => {
 };
 
 const countUnreadMessages = folders => {
-  let unreadMessageCount = 0;
   if (Array.isArray(folders?.data)) {
-    unreadMessageCount = folders.data.reduce((accumulator, currentFolder) => {
+    return folders.data.reduce((accumulator, currentFolder) => {
       return accumulator + currentFolder.attributes?.unreadCount;
     }, 0);
-  } else if (folders?.data?.attributes?.unreadCount > 0) {
-    unreadMessageCount = folders.data.attributes.unreadCount;
+  }
+  if (folders?.data?.attributes?.unreadCount > 0) {
+    return folders.data.attributes.unreadCount;
   }
 
-  return unreadMessageCount;
+  return 0;
```

Comment:
Consider reverting this change. It's generally considered good practice to have a single exit point in a function, as it can make the code easier to follow and debug